app/workers/downstream_live_worker.rb

```diff
@@ -14,8 +14,10 @@ class DownstreamLiveWorker
   def self.uniq_args(args)
     [
       args.first.fetch("content_item_id"),
+      args.first.fetch("message_queue_update_type", nil),
+      args.first.fetch("update_dependencies", true),
       'live'
```

Comment:
Probably being over zealous but would using self.name make this name more distinct (oh and should have a trailing comma)